src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportQuestionResponseResource.kt

```diff
@@ -77,7 +80,7 @@ class ReportQuestionResponseResource(
   @PreAuthorize("hasRole('ROLE_MAINTAIN_INCIDENT_REPORTS') and hasAuthority('SCOPE_write')")
   @ResponseStatus(HttpStatus.CREATED)
```

Comment:
currently still a `POST` + 201 `CREATED` request/response cycle but this will change once question code is used for unique constraints